biogtr/datasets/tracking_dataset.py

```diff
@@ -0,0 +1,143 @@
+"""Module containing Lightning module wrapper around all other datasets."""
+import torch
+from pytorch_lightning import LightningDataModule
+from typing import Union
+from torch.utils.data import DataLoader
+from biogtr.datasets.microscopy_dataset import MicroscopyDataset
+from biogtr.datasets.sleap_dataset import SleapDataset
+
+# todo: move to config
+num_workers = 0
+shuffle = True
+
+device = "cuda" if torch.cuda.is_available() else "cpu"
+
+if num_workers > 0:
+    # prevent too many open files error
```

Comment:
This logic should be handled at high level entry point - I believe this is fixed in subsequent PRs but please confirm.

Reply:
Need to remove it from this file